src/js/EditsDetailRow.jsx

```diff
@@ -1,38 +1,38 @@
 var React = require('react');
-
 var EditsDetailRow = React.createClass({
 
   propTypes: {
-    edit: React.PropTypes.object.isRequired,
+    detail: React.PropTypes.object.isRequired,
     id: React.PropTypes.number
   },
 
-  getInitialState: function(){
-    return {verification: null};
-  },
-
   componentWillMount: function(){
     this.setState({
-      verification: this.props.edit.verification
+      verification: this.props.detail.verification,
+      verified: !!this.props.detail.verification
     });
   },
 
-  makeTdContent: function(edit, field){
+  makeTdContent: function(detail, field){
     if(field === 'verification'){
-      if(this.state.verification) return this.state.verification;
-      else return <textarea onChange={this.updateText}value={this.state.verification}/>
+      if(this.state.verified) return this.state.verification;
+      else return <textarea onChange={this.updateText} value={this.state.verification}/>
     }
-    if(field === 'lar') return JSON.stringify(edit[field]);
-    return edit[field];
+    if(field === 'lar') return JSON.stringify(detail[field]);
```

Comment:
So this is actually where https://github.com/cfpb/hmda-platform-ui/issues/97 comes in, since there will be many fields but we're likely only interested in one to several per edit. I'll make the change here though as well, for great justice ;)